spec/functional/bios_get_spec.sh

```diff
@@ -68,6 +68,17 @@ End
 #  The lines of stderr should equal 0
 #End
 
+# TODO: restore when marshaling YAML errors is fixed.
```

Comment:
What errors are you hitting here?

Reply:
I actually copied this comment when I duplicated the block above, and it ended up just getting refactored to say YAML during my string replacement.

So, I don't know.

I just duplicated every `as_json` test and refactored them for YAML, inclusive of commented out tests.

Reply:
That works for now